indico_jacow/views.py

```diff
@@ -17,3 +18,7 @@ class WPDisplayAbstractsStatistics(WPJinjaMixinPlugin, WPDisplayAbstracts):
 
 class WPAbstractsStats(WPJinjaMixinPlugin, WPEventManagement):
     sidemenu_option = 'abstracts_stats'
+
+
+class WPUserMailingLists(WPJinjaMixinPlugin, WPUser):
+    sidemenu_option = 'mailing_lists'
```

Comment:
Please configure your editor to always add a trailing linebreak. In VSCode that's the `files.insertFinalNewline` setting which should be enabled.